sdk/eventhub/eventhubs-checkpointstore-blob/samples-dev/receiveEventsUsingCheckpointStore.ts

```diff
@@ -1,20 +1,14 @@
-/*
-  Copyright (c) Microsoft Corporation. All rights reserved.
-  Licensed under the MIT Licence.
-
-  This sample demonstrates how to use the EventHubConsumerClient to process events from all partitions
-  of a consumer group in an Event Hubs instance, as well as checkpointing along the way.
-
-  Checkpointing using a durable store allows your application to be more resilient. When you restart
-  your application after a crash (or an intentional stop), your application can continue consuming
-  events from where it last checkpointed.
-  
-  If your Event Hub instance doesn't have any events, then please run "sendEvents.ts" sample
-  to populate it before running this sample.
-
-  If your Event Hub instance doesn't have any events, then please run "sendEvents.ts" from the event-hubs project
-  located here: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts
-*/
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to use the EventHubConsumerClient to process events from all partitions
```

Comment:
GH won't let me comment on it, but in all the other samples we get information from environment variables instead of editing the strings in the files. The sample README template makes explicit mention of using environment variables to provide configuration. If we really want to avoid environment variables for this package, it might require some work in dev-tool to accommodate a sample that has no env var configuration.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/15137/commits/1fe41b3e6b340071861a97d13d4f948266dea2e7